### ISSUE TITLE
Question about Response Length

### ISSUE THREAD

**Issue by ayyyq** [hkust-nlp/simpleRL-reason#65]
Thanks for this excellent work.

1. I am wondering whether you have observed an increase in the `response_length/mean` logged by wandb. I assume this refers to the average of training response length. I tried 12k MATH and Qwen2.5-1.5B, and actually observed a decrease in the `response_length/mean`.
2. Did you adapt GRPO code from the original verl implementation? According to [Sea AI Lab's paper](https://arxiv.org/pdf/2503.20783), there is still a length bias in verl's GRPO implementation, and I did not see any changes made to [this function](https://github.com/volcengine/verl/blob/c6dc8b73cf011aa75b8c6a47b0322f50aed800ad/verl/trainer/ppo/core_algos.py#L301).

**Comment by Zeng-WH** [comment]
Thank you for your attention.
1. During training, our response_length/mean metric has also been increasing. Could you share your wandb logs so we can compare the trends?

2. For the most part, we followed the original verl implementation (including that function). However, we made some adjustments to other minor details in other process.

**Comment by ayyyq** [comment]
I reproduced my results once and both showed a similar trends in response_length and val accuracy (as shown below). I used Qwen2.5-1.5b, MATH-Hard as the training data, the same compute_score fn as provided, and trained for 15 epochs.

![Image](https://github.com/user-attachments/assets/094c96f9-683b-452f-a16a-a0f376b08928)

![Image](https://github.com/user-attachments/assets/faee5776-952b-4276-a9f9-716ae68d6e27)

**Comment by jiafr1802** [comment]
Did you find the reason for response_length decreasing? I had a similar trend in my experiments (but not the same task).

**Comment by ZtZhang-SCUT** [comment]
the same question.
